sdk/core/core-tracing/README.md

```diff
@@ -6,21 +6,18 @@ This is the core tracing library that provides low-level interfaces and helper m
 
 ### Installation
 
-Install this library using npm as follows
-
-```
-npm install @azure/core-tracing
-```
+This package is primarily used in Azure client libraries and not meant to be consumed directly by end users.
 
 ## Key Concepts
 
-The `@azure/core-tracing` package supports enabling tracing for Azure SDK packages, using an [OpenTelemetry](https://opentelemetry.io/) `Tracer`.
-
-By default, all libraries log with a `NoOpTracer` that takes no action. To enable tracing, you will need to set a global tracer provider following the instructions in the [OpenTelemetry getting started guide](https://opentelemetry.io/docs/js/getting_started/nodejs) or the [Enabling Tracing using OpenTelemetry example](#enabling-tracing-using-opentelemetry) below.
+- `TracingClient` is the primary interface providing tracing functionality to client libraries. Client libraries should only be aware of and interact with a `TracingClient` instance.
```

Comment:
Is `TracingClient` an interface or is it a class?

Reply:
I was a bit confused about first reading that this is an interface and then the paragraph mentions that it provides functionality and can get instanced. Should we mention that createTracingClient() gives you an instance that implements the interface?

Reply:
Ah, ok. I see what you mean. I updated the README - does it seem clearer now or do you have any thoughts on what would make sense to add?